src/openforms/variables/static_variables/static_variables.py

```diff
@@ -20,6 +20,15 @@ def get_initial_value(self, submission: Optional[Submission] = None):
         return timezone.now()
 
 
+@register_static_variable("today")
+class Today(BaseStaticVariable):
+    name = _("Today")
+    data_type = FormVariableDataTypes.date
+
+    def get_initial_value(self, submission: Optional[Submission] = None) -> str:
+        return timezone.now().date()
```

Comment:
`timezone.now()` returns a datetime with timezone information in UTC, which means if you execute this at Nov 24th 00:30 AM, the reported date will actually be Nov 23rd -> please add a unit test for that :-)

The correct way would be:

```py
now_utc = timezone.now()
timezone.localtime(now_utc).date()
```

Reply:
:see_no_evil: